scss/titles.scss

```diff
@@ -0,0 +1,23 @@
+@import 'color';
+
+.title {
+  color: $color-black;
+  font-size: 24px; 
+  position: relative;
+  left: 90px;
+  top: 60px;
+  z-index: 1;
+  
+
+  &:after {  
+  content: url(images/triangle.svg);
+  width: 90px;
+  height: 60px;
+  padding: 10px;
+  position: absolute;
+  display: inline-block;
+  top: -60px;
+  left: -80px;
```

Comment:
Try to find a positive bottom and right position instead of negative top and left